### PR TITLE
Updated test names and removed extra blank lines.

### PR DESCRIPTION
**Pull request by omaciel**

### CODE REVIEW

File: tests/foreman/api/test_contentviewfilter.py [hunk 1]
```diff
@@ -282,27 +271,25 @@ def test_create_cvf_with_same_names_negative(self):
...
 
         @Feature: Content View Filter - Create
-
         """
         with self.assertRaises(HTTPError):
             entities.RPMContentViewFilter(content_view=None).create()
 
     @run_only_on('sat')
-    def test_create_cvf_with_repo_by_id_negative(self):
+    def test_negtive_create_with_repo_by_id(self):
```
Comment (JacobCallahan) [inline tests/foreman/api/test_contentviewfilter.py]: small typo with "negative" here

File: tests/foreman/api/test_contentviewfilter.py [hunk 2]
```diff
@@ -104,7 +101,7 @@ def test_create_erratum_cvf_with_different_names(self):
                 self.assertEqual(cvf.type, 'erratum')
 
     @run_only_on('sat')
-    def test_create_pck_group_cvf_with_different_names(self):
+    def test_positive_create_pck_group_with_different_names(self):
```
Comment (sthirugn) [inline tests/foreman/api/test_contentviewfilter.py]: I think `pkg` will be more meaningful than `pck`

File: tests/foreman/api/test_contentviewfilter.py (hunk 1 above)
Comment (sthirugn) [inline tests/foreman/api/test_contentviewfilter.py]: sharp eyes!

**Comment by JacobCallahan** [comment]
ACK

**Comment by sthirugn** [comment]
ACK

**Pull request merged**